babel.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    [
+      "@babel/env"
+    ]
+  ]
+}
```

Comment:
Let's use the Typescript next time))

Reply:
ok, but I thought you said use babel at first time

Reply:
We discussed to use pure node.js at first time.

Reply:
oh, I see, sorry